.github/workflows/build-test.yml

```diff
@@ -94,7 +94,11 @@ jobs:
         export HELM_NAMESPACE=ess
         for checkov_values in charts/matrix-stack/ci/*checkov*values.yaml; do
           echo "Testing matrix-stack with $checkov_values";
-          checkov -d "charts/matrix-stack" --framework helm --var-file "$checkov_values" --quiet
+          tmpdir=$(mktemp -d)
+          cd "$tmpdir"
+          helm template checkov -f "../$checkov_values" ../charts/matrix-stack | sed 's/{{//' | yq -s '.kind + "-" + .metadata.name'
+          checkov -d .  --skip-check CKV_SECRET_6 --quiet
```

Comment:
We should be using the script here rather than implementing things twice

Reply:
:facepalm: